src/nodes.coffee

```diff
@@ -1456,6 +1461,18 @@ exports.Class = class Class extends Base
 
     ctor
 
+  proxyBoundMethods: (o) ->
+    @variable ?= new IdentifierLiteral o.scope.freeVariable '_class' if @parent
+    [@variable, @variableRef] = @variable.cache o unless @variableRef?
+
+    @ctor.thisAssignments = for method in @boundMethods by -1
```

Comment:
Nitpick: I know the `by -1` was restored from the old version, but I suspect it must be from a time when the expressions were being `unshift`ed directly into the function body. Currently they are being assigned in reverse source order (not that it matters, besides aesthetics). Would be nice to fix this :smile: 

```coffee
class extends Foo
  a: =>
  b: =>
```
```js
_class = (class extends B {
  constructor () {
    super(...arguments)
    this.b = ...
    this.a = ...
  }
})
```